lib/components/viewers/stop-time-cell.tsx

```diff
@@ -1,9 +1,6 @@
 import { format, getTimezoneOffset, utcToZonedTime } from 'date-fns-tz'
-import { FormattedMessage } from 'react-intl'
+import { FormattedMessage, FormattedTime } from 'react-intl'
```

Comment:
Unused import

Reply:
Good catch thanks! Removed in 8ebe1c125d977bc084055d622cd429e24bbd4c05